MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1110,7 +1113,26 @@ private void SetAudioInfoFromTags(MediaInfo audio, IReadOnlyDictionary<string, s
                 }
             }
 
-            // Check for writer some music is tagged that way as alternative to composer/lyricist
+            if (tags.TryGetValue("performer", out var performer) && !string.IsNullOrWhiteSpace(performer))
+            {
+                foreach (var person in Split(performer, false))
+                {
+                    Match match = _performerPattern.Match(person);
+
+                    // If the performer doesn't have any instrument/role associated, it won't match. In that case, chances are it's simply a band name, so we skip it.
+                    if (match.Success)
+                    {
+                        people.Add(new BaseItemPerson
+                        {
+                            Name = match.Groups["name"].Value,
+                            Type = PersonType.Actor,
+                            Role = CultureInfo.CurrentCulture.TextInfo.ToTitleCase(match.Groups["instrument"].Value)
```

Comment:
Personally, I'd leave it like it is

Reply:
It's lowercase in what Picard puts in, at least. So it'd look a bit out of place in clients.